sdk/eventhub/Azure.Messaging.EventHubs/src/EventPosition.cs

```diff
@@ -112,18 +112,18 @@ public sealed class EventPosition
 
         /// <summary>
         ///   Corresponds to a specific date and time within the partition to begin seeking an event; the event enqueued after the
-        ///   requested <paramref name="enqueuedTimeUtc" /> will become the current position.
+        ///   requested <paramref name="enqueuedTime" /> will become the current position.
         /// </summary>
         ///
-        /// <param name="enqueuedTimeUtc">The date and time, in UTC, from which the next available event should be chosen.</param>
+        /// <param name="enqueuedTime">The date and time from which the next available event should be chosen.</param>
```

Comment:
Let's please keep the ", in UTC," call outs in the comments.

Reply:
Done.